src/Lean/Compiler/IR/EmitLLVM.lean

```diff
@@ -1374,6 +1374,15 @@ def callLeanInitialize (builder : LLVM.Builder llvmctx) : M llvmctx Unit := do
   let fn ← getOrCreateFunctionPrototype (← getLLVMModule) retty fnName argtys
   let _ ← LLVM.buildCall2 builder fnty fn #[]
 
+def callLeanSetupLibUV (builder : LLVM.Builder llvmctx) (argc argv : LLVM.Value llvmctx) : M llvmctx (LLVM.Value llvmctx) := do
+  let fnName :=  "lean_setup_libuv"
+  let intTy ← LLVM.intTypeInContext llvmctx 64
```

Comment:
If this were a 64-bit type, couldn't this just be `LLVM.i64Type`? But shouldn't this be 32-bit to match the C `int`?

Reply:
Fixed by adding a trunc and changing the type to i32